src/console.cpp

```diff
@@ -236,7 +236,7 @@ void IConsoleAddSorted(T **base, T *item_new)
 /**
  * Remove underscores from a string; the string will be modified!
  * @param name The string to remove the underscores from.
- * @return #name.
+ * @return the modified string.
```

Comment:
Well, it *does* return the input pointer unmodified. But maybe `@param name` should be `@param[inout] name` to indicate that the pointed-to string data is modified?